talisker/logs.py

```diff
@@ -351,6 +351,10 @@ def logfmt_atom(self, k, v):
                 v = v[:size] + self.TRUNCATED
 
             v = '"' + v + '"'
+        elif isinstance(v, bool):
+            v = str(v).lower()
+        elif not isinstance(v, int):
+            v = '"' + str(v).replace('"', '') + '"'
```

Comment:
I think this would be better defined recursively

`return self.logfmt_atom(self, k, str(v))`

otherwise you risk hitting cases that you account for with the string handling e.g. blowing the size